### PR TITLE
#5328: sliding window indices tensor need to be page size aligned

### PR DESCRIPTION
**Pull request by nsmithtt**

### CODE REVIEW

**Comment by nsmithtt** [comment]
I left @shwetankTT's branch intact for now to separate the ttnn test port + other failure signatures from the test list, i.e. cases that are using an invalid sharding strategy.

**Review by mywoodstock — APPROVED** [review]
👍🏾

File: tests/tt_eager/python_api_testing/unit_testing/test_optimized_conv_v2.py [hunk 1]
```diff
@@ -48,6 +48,7 @@
         # Small conv
         (1, 32, 32, 16, 16, 3, 3, 2, 2, 1, 1, True),
         # (1, 32, 32, 16, 16, 3, 3, 2, 2, 1, 1, False), # Asserts #5323
+        (2, 256, 128, 3, 3, 3, 3, 1, 1, 0, 0, True),
```
Comment (tt-nshanker) [inline tests/tt_eager/python_api_testing/unit_testing/test_optimized_conv_v2.py:51]: Please move this test to `test_conv.py. We want to deprecate this test file.

**Review by tt-nshanker — CHANGES_REQUESTED** [review]

File: tests/tt_eager/python_api_testing/unit_testing/test_optimized_conv_v2.py (hunk 1 above)
Comment (nsmithtt) [reply tests/tt_eager/python_api_testing/unit_testing/test_optimized_conv_v2.py]: @tt-nshanker, done! Moved this test to the ttnn version

**Review by tt-nshanker — APPROVED** [review]

**Pull request merged**